python/mujincontrollerclient/realtimerobotclient.py

```diff
@@ -822,19 +822,14 @@ def SetRobotBridgeIOVariables(self, iovalues, robotname=None, timeout=10, useweb
         return self.ExecuteCommand(taskparameters, robotname=robotname, timeout=timeout, usewebapi=usewebapi)
     
     def ComputeIkParamPosition(self, name, robotname=None, timeout=10, usewebapi=None, **kwargs):
-        """
+        """Given the name of a Kinbody, computes the manipulator (TCP) position in the kinbody frame to generate values for an IKParameterization.
 
         Args:
-            name (str):
-            robotname (str, optional): Name of the robot
-            timeout (float, optional):  (Default: 10)
+            name (str): Name of the Kinbody (the robot).
+            robotname (str, optional): (unused) Name of the robot.
```

Comment:
@felixvd it is a bit hard to follow in the code, but I think ``robotname`` is used.

Reply:
Right. It's redundant, but `SetGoals` uses it to set up the environment.